pfetch

```diff
@@ -246,6 +246,13 @@ get_os() {
             trap '' EXIT
         ;;
 
+        SunOS)
+            IFS='(' read -r distro _ < /etc/release
+            kstat=$(kstat -p unix:0:system_misc:snaptime \
```

Comment:
I was thinking more along the lines of adding the parser here *once*.

The `case` statement in the loop to populate `pages_free`, `pages_total`, and `seconds` (can be further stripped of `.` in `get_uptime`).